content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
+
+    1.  Choose Public as your *Zone Type*.
+
+    2.  Type a *Zone Name* for your zone.
+
+    3.  Input a DNS suffix in *DNS name*, for example
+        `acmecorp.example`.
+
+    4.  Choose your *DNSSEC* or DNS Security state, which should be set
+        to **Off** for this configuration.
+
+    5.  (Optional) Input a *Description* for your DNS zone.
+
+    6.  Click **Create**.
+
+Once created, the *Zone Details* page loads. *NS* (Name server) and
+*SOA* (Start of autority) records are automatically created for your
+domain (for example `acmecorp.example`)
+
+# Configuring External DNS in Jenkins X
+
+Once you have configured Google Cloud DNS, you can use browse the
+[Zones](https://console.cloud.google.com/net-services/dns/zones) page in
+your Google Cloud Platform project to setup your external domain.
+
+> **Note**
+>
+>External DNS will automatically updates DNS records if you reuse the domain
+>name, so if you delete an old cluster and create a new one it will preserve the
+>same domain configuration for the new cluster.
+
+1.  Choose a unique DNS name; you can use nested domains (for example,
+     `cluster1.acmecorp.example`). Enter the name in the `DNS Name`
+    field
+
+2.  Run the `jx create domain` command against
+    `jx create domain gke --domain cluster1.acmecorp.example`.
+
+    1.  The program prompts you to choose your Google Cloud Platform
+        project from the available list.
+
+    2.  The program prompts you to update your existing managed servers
+        to use the displayed list of Cloud DNS nameservers. Copy the
+        list for use in the next steps.
+
+3.  From the Google Cloud Platform
+    [Zones](https://console.cloud.google.com/net-services/dns/zones)
+    page, change the *Resource Record Type* to `NS`) and use the default
+    values for your domain for for *TTL* (`5`) and *TTL Unit*
+    (`minutes`).
+
+4.  Add the first nameserver to the *Name server* field
+
+5.  Click **Add item** and add any subsequent nameservers.
+
+6.  Click **Create**.
+
+7.  Configure Jenkins X for the new domain names:
+
+    -   From the directory where you extracted the Jenkins X tarball
+        archive open the `jx-requirements-gke.yml` file in a text editor
+        (such as TextEdit for macOS or gedit for Linux) and edit the
+        ingress section at the root level.
+
+            ingress:
+              domain: cluster1.acmecorp.example
+              externalDNS: true
+              namespaceSubDomain: -jx.
+              tls:
+                email: certifiable@acmecorp.example
+                enabled: true
+                production: true
+
+# Installing Jenkins X on your GKE cluster
+
+You are now ready to install Jenkins X to your cluster. From your
+command-line, change to the directory containing the pre-configured
+`jx-requirements-gke.yml` requirements file that was unarchived with
+your `jx` program binary. For example, macOS defaults to the `Downloads`
+directory:
+
+    cd /Users/acmeuser/Downloads/
```

Comment:
Try to make the commands executable without changing them (when possible). That command would be better written as `cd ~/Downloads/`.